apps/prairielearn/src/models/assessment.ts

```diff
@@ -32,3 +45,41 @@ export async function selectAssessmentInfoForJob(assessment_id: string) {
     }),
   );
 }
+
+export const AssessmentStatsRowSchema = AssessmentSchema.extend({
+  needs_statistics_update: z.boolean().optional(),
+});
+export type AssessmentStatsRow = z.infer<typeof AssessmentStatsRowSchema>;
```

Comment:
Nit: is this type ever used in isolation? If not, I'd rather we collapse this into the `AssessmentRowSchema` and type below.

Reply:
It is, it is also used by the assessment stats that have their own separate endpoint and load asynchronously.